src/core/Tokenizer.js

```diff
@@ -12,15 +12,17 @@ export default class Tokenizer {
      *  @param {String[]} cfg.closeParens Closing parentheses to enable, like ), ]
      *  @param {String[]} cfg.indexedPlaceholderTypes Prefixes for indexed placeholders, like ?
      *  @param {String[]} cfg.namedPlaceholderTypes Prefixes for named placeholders, like @ and :
+     *  @param {String[]} cfg.lineCommentTypes Line comments to enable, like # and --
      */
     constructor(cfg) {
-        this.WORD_REGEX = /^(\w+)/;
+        this.WORD_REGEX = /^([\w|#|@]+)/;
```

Comment:
I tried playing around with word and operators tokenizing order (match operators before words) but that was no-go. Currently I went with the easiest solution that shouldn't break anything...